lost/src/test/java/com/mapzen/android/lost/internal/LostApiClientImplTest.java

```diff
@@ -65,7 +65,7 @@ public void disconnect_shouldRemoveGeofencingApiImpl() throws Exception {
     }
 
     @Test
-    public void disconnect_shouldUnregisterLocationUpdateListeners() throws Exception {
+    public void disconnect_shouldNotUnregisterLocationUpdateListeners() throws Exception {
```

Comment:
I had thought it would be good to enforce best practices on the user of this api but there isn't any harm in removing all listeners and shutting down as you suggest. I'll update the code.